api-reference/docstubs/api/microsoft.fast.pi.viewgenerator.v2.profile.models-user-post-user.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Create user"
```

Comment:
Curious why we generated the POST and PATCH topics for the user resource?

Then what about the CRUD topics for the new regionalAndLanguageSettings entity?

Reply:
Yes - I already made that comment in the meeting chat.  I still need to file a bug.  Generating the resource file is fine, but the CRUD API topics - not so much (when this CSDL is not the master for the object)

Reply:
Oh, I think dkershaw was responding to not generating the CRUD operation topics for the _user_ entity, and that the input CSDL is not the master for the _user_ entity. Agree with that.

My next question was about the CRUD operations for the new regionalAndLanguageSettings entity - why are they not generated?
The input CSDL is the master CSDL for that new entity.